mkosi/__init__.py

```diff
@@ -7720,11 +7724,13 @@ def find_address(args: MkosiArgs) -> Tuple[str, str]:
 def run_command_image(args: MkosiArgs, commands: Sequence[str], timeout: int, check: bool, stdout: _FILE = sys.stdout, stderr: _FILE = sys.stderr) -> CompletedProcess:
     if args.verb == Verb.qemu:
         return run_ssh(args, commands, check, stdout, stderr, timeout)
-    else:
+    elif args.verb == Verb.boot:
         cmdline = ["systemd-run", "--quiet", "--wait", "--pipe", "-M", virt_name(args), "/usr/bin/env", *commands]
         with suppress_stacktrace():
             return run(cmdline, check=check, stdout=stdout, stderr=stderr, text=True, timeout=timeout)
-
+    else:
+        with suppress_stacktrace():
+            return run(run_shell_cmdline(args, True, commands), check=check, stdout=stdout, stderr=stderr, text=True, timeout=timeout)
```

Comment:
Just passing True is a bit unreadable, let's add the keyword here as well to explain things `pipe=True`